otel-extension/src/main/java/io/scalac/mesmer/otelextension/MesmerAkkaPersistenceInstrumentationModule.java

```diff
@@ -1,20 +1,20 @@
 package io.scalac.mesmer.otelextension;
 
 import com.google.auto.service.AutoService;
-import com.typesafe.config.ConfigFactory;
 import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
 import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
 import io.scalac.mesmer.agent.akka.persistence.AkkaPersistenceAgent;
+
 import java.util.List;
 
 @AutoService(InstrumentationModule.class)
 public class MesmerAkkaPersistenceInstrumentationModule extends InstrumentationModule {
-  public MesmerAkkaPersistenceInstrumentationModule() {
-    super("mesmer-akka-persistence");
-  }
+    public MesmerAkkaPersistenceInstrumentationModule() {
```

Comment:
are you using "google java format" in intelij? There's a plugin for that to make things simpler. When I was alone in the project, I kinda enforced that formatting. 🙈  Pre-commit also uses that and has good support for it. 

lmk if you have objections.

Reply:
Yeah, I use IntelliJ one, but I don't have strong feelings about java formatting - we can use formatting chosen by you :)